activiti-cloud-services-query/activiti-cloud-services-query-graphql/src/main/java/org/activiti/cloud/services/query/graphql/autoconfigure/ActivitiGraphQLAutoConfiguration.java

```diff
@@ -43,7 +43,7 @@
  */
 @Configuration
 @ConditionalOnClass(GraphQL.class)
-@ConditionalOnProperty(name = "spring.activiti.cloud.services.query.graphql.enabled")
+@ConditionalOnProperty(name = "spring.activiti.cloud.services.query.graphql.enabled", matchIfMissing = true)
```

Comment:
Is that normal that there is only `@ConditionalOnProperty` and not `@ConditionalOnExpression` here?